apps/src/code-studio/pd/workshop_dashboard/components/survey_results/survey_rollup_table.jsx

```diff
@@ -95,6 +105,12 @@ export class SurveyRollupTable extends React.Component {
     }));
   }
 
+  renderAverage(value, category) {
+    return value
+      ? `${value.toFixed(1)} / ${this.categoryDenominator[category]}`
```

Comment:
Should this function handle the case where we haven't hard-coded a denominator for the given category?

Reply:
I guess we can show a question mark `?` for missing denominator. 
However, that case shouldn't happen since we tightly control what categories used to calculate roll-up results. The server shouldn't return data that the client doesn't expect.